src/domcoords.js

```diff
@@ -40,6 +41,7 @@ exports.scrollRectIntoView = scrollRectIntoView
 // when the size of the content above changes.
 function storeScrollPos(view) {
   let rect = view.dom.getBoundingClientRect(), startY = Math.max(0, rect.top)
+  let doc = view.dom.defaultView
```

Comment:
This looks like it'll yield `undefined`, and definitely not the document object (since that would be `ownerDocument`). Am I missing something?